phys2bids/interfaces/acq.py

```diff
@@ -12,17 +12,14 @@
 LGR = logging.getLogger(__name__)
 
 
-def populate_phys_input(filename, chtrig):
+def populate_phys_input(filename, chtrig=0):
```

Comment:
If `chtrig` isn't being used anymore (and based on the docstring change that's what I'm inferring), then why keep the argument?

Reply:
basically chtrig is called at the end of the function by ``populate_phys_auto``

Reply:
Ah, thank you. It needs to stay in the docstring then.